modules/bringauto_repository/GitLFSRepository.go

```
@@ -43,40 +50,155 @@ func (lfs *GitLFSRepository) CheckPrerequisites(*bringauto_prerequisites.Args) e
+				break
+			}
+			logger.WarnIndent(expectedPackForImagePath)
+		}
+	}
+	if len(expectedPackNotForImagePaths) > 0 {
+		logger.Warn("%d packages are in context but are not built for target image (listing first %d):", len(expectedPackNotForImagePaths), listFileCount)
+		for i, expectedPackNotForImagePath := range expectedPackNotForImagePaths {
+			if i > listFileCount - 1 {
+				break
+			}
+			logger.WarnIndent(expectedPackNotForImagePath)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to (*bringauto/modules/bringauto_log.Logger).WarnIndent

(govet)